### DISCUSSION TITLE
How to use new s3 server feature, need a guide

### DISCUSSION THREAD

**Discussion by jibwf** [AlistGo/alist#6189] [General]
如何使用新版本的s3 server功能啊？客户端要求填写endpoint, region, access key, secure key. 设置中只找到access key 和secure key

**Comment by jibwf** [comment]
@itsHenry35

**Comment by TinyInstallerCode** [comment]
+1 seem it's not working

**Comment by itsHenry35** [comment]
还有个auth需要修复，不急，等全都修好了会更新docs的，先当个dev版用用吧
